docs/assets/scss/_td-content.scss

```
@@ -1,6 +1,9 @@
 // FONT styles in td-content section
 
 .td-content {
+  overflow-wrap: break-word;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)